src/Makefile

```diff
@@ -69,12 +69,14 @@ ifeq ($(PREFIX),)
     PREFIX := /usr
 endif
 
+override LUAPATH ?= $(PREFIX)/lib/lua
```

Comment:
I think it wouldn't work as we probably expected.
If we really want to override the variable that might be set with a command argument, e.g:
`LUAPATH=$PATH make -f src/Makefile`
we need to write: 
`override LUAPATH = $(PREFIX)/lib/lua`
or
`override LUAPATH := $(PREFIX)/lib/lua`

Reply:
I assume here intention was to define the default value using `?=`, so override is meaningless and statement should look lke:
`LUAPATH ?= $(PREFIX)/lib/lua`

Reply:
fixed